tests/server/sws.c

```diff
@@ -1894,7 +1894,7 @@ static int service_connection(curl_socket_t msgsock, struct httprequest *req,
   while(!req->done_processing) {
     int rc = get_request(msgsock, req);
     if(rc <= 0) {
-      /* Nothing further to read now (possibly because the socket was closed */
+      /* Nothing further to read now (possibly because the socket was closed) */
```

Comment:
The Travis job is failing because this line is now longer than 79 columns. Could you please fix that? Maybe just replace the parentheses with a comma or something or use two lines for the comment.